modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4922,6 +4922,35 @@ public void testGetObjectEntries() throws Exception {
 			).build());
 	}
 
+	@FeatureFlag("LPD-53981")
+	@Test
+	public void testGetObjectEntriesTotalCountMatchesItemCount()
+		throws Exception {
+
+		ObjectEntry objectEntry1 = _addObjectEntry(
+			_objectDefinition4, _group.getGroupKey(), 1);
+
+		_defaultObjectEntryManager.deleteObjectEntry(
+			_simpleDTOConverterContext, _objectDefinition4,
+			objectEntry1.getId());
+
+		ObjectEntry objectEntry2 = _addObjectEntry(
+			_objectDefinition4, _group.getGroupKey(), 1);
+
+		Page<ObjectEntry> expectedPage = Page.of(
+			Collections.singletonList(objectEntry2));
+
+		Page<ObjectEntry> actualPage = _objectEntryManager.getObjectEntries(
+			TestPropsValues.getCompanyId(), _objectDefinition4,
+			_group.getGroupKey(), null, _createDTOConverterContext(),
+			(String)null, null, null, null);
+
+		assertEquals(actualPage, expectedPage);
+
+		Assert.assertEquals(
```

Comment:
Should this logic be inside method `protected void assertEquals(Page<ObjectEntry> actualPage, Page<ObjectEntry> expectedPage)`?